docs: clarify steps to verify node app in VM (#652)
canonical/rockcraft: docs/tutorial/node-app.rst

    2023-10-30T12:37:33.663Z [pebble] Service "app" starting: node server.js
    2023-10-30T12:37:33.864Z [app] Running on http://0.0.0.0:8080

> 

Author: https://canonical-ubuntu-documentation-library--652.com.readthedocs.build/rockcraft/en/652/tutorial/node-app/#run-the-rock-in-docker

    2023-10-30T12:37:33.663Z [pebble] Service "app" starting: node server.js
    2023-10-30T12:37:33.864Z [app] Running on http://0.0.0.0:8080

> Next we'll verify that the Node.js app is up and running. If you're running this
> tutorial on a regular Ubuntu system, open your web browser and navigate to

Reviewer: ```suggestion
Next we'll verify that the Node.js app is up and running. If you're working on
a regular Ubuntu system, open your web browser and go to
```

A bit of a nitpick, but "run" isn't the best verb to apply to tutorials.

I find "navigate" to be cumbersome and prefer "go", but I'll leave it up to you to choose.

``http://localhost:8000``. You should see a blank page with a
"Hello World from inside the rock!" message. Success!

> If, instead, you are following the tutorial through a Multipass VM, you can
> open another shell into the VM and access the app using curl:

Reviewer: ```suggestion
If, instead, you're working in a Multipass VM, you can
open another shell into the VM and access the app with curl:
```

Simple contractions with pronouns improve readability ("you are" → "you're").

I try and cut down on the instances of "use" and "using" to avoid repetition and make the phrase's core verb (`access`) more prominent, but YMMV.

Author: makes sense!

.. literalinclude:: code/node-app/task.yaml
    :language: bash
    :prepend: multipass shell rock-dev
    :start-after: [docs:curl-localhost]
    :end-before: [docs:curl-localhost-end]
    :dedent: 2

> This should also print "Hello World from inside the rock!" to the terminal.

Reviewer: ```suggestion
If successful, the output is "Hello World from inside the rock!".
```

We want to avoid instances of non-specific "this".
